azure-stack/hci/manage/manage-syslog-forwarding.md

```diff
@@ -0,0 +1,469 @@
+```
+
+#### Cmdlet parameters
+
+The following table provides parameters for the `Set-AzSSyslogForwarder` cmdlet:
+
+|Parameter |Description |Type |Required |
+|-----|-----|------|-----|
+|ServerName |FQDN or IP address of the syslog server. |String |Yes |
+|ServerPort |Port number the syslog server is listening on. |UInt16 |Yes |
+|NoEncryption |Force the client to send syslog messages in clear text. |Flag |No |
+|SkipServerCertificateCheck |Skip validation of the certificate provided by the syslog server during initial TLS handshake. |Flag |No |
+|SkipServerCNCheck |Skip validation of the Common Name value of the certificate provided by the syslog server during initial TLS handshake. |Flag |No |
+|UseUDP |Use syslog with UDP as transport protocol. |Flag |No |
+|ClientCertificateThumbprint |Thumbprint of the client certificate used to communicate with syslog server. |String |No |
+|OutputSeverity |Level of output logging. Values are Default or Verbose. Default includes severity levels: warning, critical, or error. Verbose includes all severity levels: verbose, informational, warning, critical, or error. |String |No |
+|Remove |Remove current syslog forwarder configuration and stop syslog forwarder. |Flag |No |
+
+### Syslog forwarding with TCP, mutual authentication (client and server), and TLS 1.2 encryption
+
+In this configuration, the syslog client in Azure Stack HCI forwards messages to the syslog server over TCP using TLS 1.2 encryption. During the initial handshake, the client verifies that the server provides a valid, trusted certificate. The client also provides a certificate to the server as proof of its identity.
+
+This configuration is the most secure as it provides a full validation of the identity of both the client and the server, and it sends messages over an encrypted channel.
+
+> [!IMPORTANT]
+> Microsoft strongly recommends that you use this configuration for production environments.
+
+To configure syslog forwarder with TCP, mutual authentication, and TLS 1.2 encryption, run both these cmdlets against a physical host:
+
+Configure the server and provide certificate to the client to authenticate against the server. Run the following command:
+
+   ```azurepowershell
+   Set-AzSSyslogForwarder -ServerName <FQDN or ip address of syslog server> -ServerPort <Port number on which the syslog server is listening on> -ClientCertificateThumbprint <Thumbprint of the client certificate>
+   ```
+
+> [!IMPORTANT]
+> The client certificate must contain a private key. If the client certificate is signed using a self-signed root certificate, you must import the root certificate as well.
+
+Here is an example to set and import certificates for mutual authentication.
+
+This example script must be run from a deployment virtual machine (DVM).
+
```

Comment:
Just check against Azure docs if DVM is an approved acronym!

Reply:
Done > presented as "the Deployment VM (DVM)" on first mention, then DVM.